apps/condo/domains/common/components/LayoutContext.tsx

```diff
@@ -9,6 +9,7 @@ const { useBreakpoint } = Grid
 interface ILayoutContext {
     isMobile?: boolean
     isSmall?: boolean
+    lowResScroll?: boolean
```

Comment:
This variable is defined within `LayoutContext`, so it's good to know without debugging what this variable using for.
Also, this is a boolean variable. A good practice to name such variables starting with `is`, `has`, `should`.
As for this particular one, I suggest `shouldTableScroll`.